src/effects/VST3/VST3UIValidator.cpp

```diff
@@ -166,16 +166,31 @@ void VST3UIValidator::OnClose()
 
    mWrapper.EndParameterEdit();
 
+   //Make sure all previous changes has been processed by the worker
+   mAccess.Flush();
```

Comment:
Maybe  not needed with idle time flushes now

Reply:
Disagree. This flush happens at a very specific time. Removing it may result in rare random bugs